.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: nomaddo/cross-rpi
+    steps:
+      - checkout
+      - run:
+          name: download header
+          command: git clone https://github.com/doe300/VC4CLStdLib.git VC4CLStdLib
+      - run:
+          name: get libVC4CC.so url
+          command: curl "https://circleci.com/api/v1.1/project/github/doe300/VC4C/latest/artifacts?circle-token=${CIRCLECI_API_TOKEN}=master&filter=successful" --output /tmp/dump
+      - run:
+          name: get libVC4CC.so
+          command: wget -O libVC4CC.so $(python tools/get_url.py "/tmp/dump")
+      - run:
+          name: configure
+          command: cmake . -DCROSS_COMPILE=ON -DBUILD_TESTING=ON -DCMAKE_FIND_DEBUG_MODE=1  -DOpenCL_INCLUDE_DIR=/opt/rasberrypi/tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/arm-linux-gnueabihf/include -DOpenCL_LIBRARY=/opt/rasberrypi/tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/arm-linux-gnueabihf/include -DCMAKE_INSTALL_PREFIX=/usr -DVC4CL_STDLIB_HEADER_SOURCE=./VC4CLStdLib/include/VC4CLStdLib.h -DVC4CC_LIBRARY=libVC4CC.so
```

Comment:
We install VC4CLStdLib and VC4CL into `/usr/local/`, shouldn't we then also install VC4CL into `/usr/local`?

Reply:
Another thing: For VC4CL to find VC4C, the library and the headers need to be found. Wouldn't it be easier to download vc4c-xxx.deb, install it and use its headers and library?
I changed CMakeLists.txt, so it should automatically find the library and headers in the installed directories.